KMC/kmc.hpp

```diff
@@ -1,9 +1,9 @@
 #ifndef KMC_HPP
 #define KMC_HPP
 
-#include "KMC/helpers.hpp"
```

Comment:
KMC/* is used for submodule purposes with Wen's code. This allowed for easier catch2 testing and prevent cases were people may have other helper/integration/macros/etc. header files